pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -201,8 +201,14 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	}
 	allErrors = append(allErrors, metrics.ValidateShowHiddenMetricsVersion(kc.ShowHiddenMetricsForVersion)...)
 
+	return utilerrors.NewAggregate(allErrors)
+}
+
+// ValidateKubeletLoggingConfiguration validate kubelet logging part config
```

Comment:
Doesn't look like this function is needed. just use `logs.ValidateLoggingConfiguration(&kc.Logging, field.NewPath("logging").ToAggregate()`